app/models/language.rb

```diff
@@ -2,44 +2,12 @@
 #
 # Table name: languages
 #
+#  created_at :datetime
 #  id         :integer          not null, primary key
+#  locale     :string
 #  name       :string
-#  code       :string
-#  created_at :datetime
 #  updated_at :datetime
 #
 
 class Language < ActiveRecord::Base
```

Comment:
What I do in ELMO is just have an entry in the translation file called 'locale_name'. In en.yml it's 'English', in fr.yml it's 'Français', etc. Then you can grab them all by iterating over the locales you have defined and doing `t('locale_name', locale: l)`